src/main/java/com/kryptnostic/kindling/search/ElasticsearchTransportClientFactory.java

```diff
@@ -0,0 +1,92 @@
+package com.kryptnostic.kindling.search;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+
+import org.elasticsearch.action.admin.cluster.state.ClusterStateRequest;
+import org.elasticsearch.action.admin.cluster.state.ClusterStateResponse;
+import org.elasticsearch.client.Client;
+import org.elasticsearch.client.node.NodeClient;
+import org.elasticsearch.client.transport.TransportClient;
+import org.elasticsearch.cluster.node.DiscoveryNode;
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.common.transport.InetSocketTransportAddress;
+import org.elasticsearch.transport.client.PreBuiltTransportClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.collect.ImmutableList;
+
+public class ElasticsearchTransportClientFactory {
+	
+	public static final Logger log = LoggerFactory.getLogger( ElasticsearchTransportClientFactory.class );
```

Comment:
We use logger pretty much everywhere, including earlier in this PR. Switch to logger here for consistency.